src/Plugin/GraphQL/Fields/FieldPluginBase.php

```diff
@@ -132,7 +152,9 @@ protected function isLanguageAwareField() {
    * {@inheritdoc}
    */
   protected function resolveDeferred(callable $callback, $value, array $args, ResolveContext $context, ResolveInfo $info) {
-    $result = $callback($value, $args, $context, $info);
+    $result = $this->getRenderer()->executeInRenderContext(new RenderContext(), function () use ($callback, $value, $args, $context, $info, &$result) {
```

Comment:
What does the &$result parameter in use() do there?

Reply:
Nothing.. removed.